TrafficCapture/dockerSolution/src/main/docker/migrationConsole/lib/console_link/README.md

```diff
@@ -0,0 +1,131 @@
+# Console_link Library
+
+The console link library is designed to provide a unified interface for the many possible backend services involved in a migration. The interface can be used by multiple frontends--a CLI app and a web API, for instance.
+
+![Console_link Library Diagram](console_library_diagram.svg)
```

Comment:
Lets not checking images, but instead use inline renderable mermaid graphs, here is one I made for this page:
```suggestion
```mermaid
graph LR
    CLI["Command line interface"] --> Library["Console link Library"]
    Web_API["Programmatic REST API"] --> Library
    Library --> Source["Source Cluster"]
    Library --> Target["Target Cluster"]
    Library --> Services["Cloud Services"]
`` ` <-- Delete this space
```

#### Here is how that renders:

```mermaid
graph LR
    CLI["Command line interface"] --> Library["Console link Library"]
    Web_API["Programmatic REST API"] --> Library
    Library --> Source["Source Cluster"]
    Library --> Target["Target Cluster"]
    Library --> Services["Cloud Services"]
```

Reply:
+1 when it's possible.  We have some scenarios where it isn't feasible (like our architecture document).
But how does one get that to render?  In my markdown reader (IntelliJ), that doesn't work

Reply:
You use intellij right? - https://www.jetbrains.com/help/idea/markdown.html
For VSCode folks like me - https://marketplace.visualstudio.com/items?itemName=bierner.markdown-mermaid

While not in an IDE [mermaid-live-editor](https://mermaid-js.github.io/mermaid-live-editor/edit#pako:eNp9kU1qwzAQha8iZp1cwItCcbMouBDiQBdVKRNpnIjqJ4ylQgi5e8dWHUoX1Wr09L2n0egKJlmCBo6M55PqdjoqWW33_KahTSFgtMq7SMrFTDygIQ3var1-UJ07MPJl5uKYPE3c5yILVaNe6fDxuJ3itpzkFonMzqjdpt8r0f-kVc_PZj7oU2FDYq-Fan0ZpZN7_m92j3ykLGwt_mV74i9naJz696nYuyA0rCAQB3RWJnOdvBryiYK8vZHS0oDFZw063gQtZ4uZNtblxNAM6EdaAZac-ks00GQutEBPDqcRLCLNnpf6A_NH3L4BGhOFWA) is pretty good.

Alternatively opening any GitHub comment windows preview will also generate the chart if you need to quickly write something to copy paste out